config/locales/pt-BR.yml

```diff
@@ -86,6 +86,9 @@ pt-BR:
       reject: "rejeitar"
       restart: "reiniciar"
       clone: "Clonar história"
+      copy_url: "Copiar URL"
+      copy_id: "Copiar ID"
+      look_history: "Ver história"
```

Comment:
Maybe it would be better to change it to `Ver histórico` since the `clone` key already has `história` in it with a different meaning. What do you think @andrerpbts?